src/main/java/gui/QualityRulesResultFrame.java

```diff
@@ -75,6 +86,59 @@ private void updateCentralPanel() {
 		centralPanel.add(tableScrollPane);
 		qualityFrame.add(centralPanel, BorderLayout.CENTER);
 	}
+	
+	/**
+	 * Updates the quality indicators panel
+	 */
+	private void updateSouthPanel() {
```

Comment:
can we have one method that calls this and the updateCentralPanel() so that the onSave() and onDelete() action listeners from the EditRuleController can call it?